apps/web/pages/signup.tsx

```diff
@@ -402,10 +398,14 @@ export default function Signup({
                         return;
                       }
                       const username = formMethods.getValues("username");
+                      if (!username) {
```

Comment:
Because there is a theoretical possibility of username being null in the flow. Show the error appropriately if it happens.